test/bench.cpp

```diff
@@ -4,87 +4,370 @@
 #include <string>
 #include <vector>
 
+#include <sstream>
+
+#include <stdio.h>
+#include <stdbool.h>
+
+#include <SDL.h>
+#include <SDL_ttf.h>
+#include <cstdio>
+#include <ctype.h>
```

Comment:
C headers (`*.h`) are present in C++ with names like `c*`, e.g. `cstdio` for `stdio.h`: just for the sake of consistency use one form or another, but not the both. Avoid duplication of header inclusion -- again, for consistency.